test/quantization/core/test_quantized_op.py

```diff
@@ -1066,6 +1066,29 @@ def test_channel_shuffle(self, X, groups):
         self.assertEqual(a_ref, a_hat.dequantize(),
                          msg="torch.nn.functional.channel_shuffle results are off")
 
+    """Tests pixel shuffle operation on quantized tensors."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=4, max_dims=4,
```

Comment:
please use simple loops to choose between different options, or just hardcode these parameters, we plan to move away from hypothesis, example: https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L439

Reply:
The test case has been fixed, hypothesis is removed.